### PR TITLE
To avoid I/O errors, carry out vg deactivate (using vgchange -an)

### PR DESCRIPTION
**Pull request by SaravanaStorageNetwork**
and dmsetup remove device.

Signed-off-by: Saravanakumar Arumugam <sarumuga@redhat.com>

### CODE REVIEW

File: roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml [hunk 1]
```diff
@@ -128,7 +128,7 @@
 
   # Runs "lvremove -ff <vg>; vgremove -fy <vg>; pvremove -fy <pv>" for every device found to be a physical volume.
 - name: Clear GlusterFS storage device contents
-  shell: "{% for line in item.stdout_lines %}{% set fields = line.split() %}{% if fields | count > 1 %}lvremove -ff {{ fields[1] }}; vgremove -fy {{ fields[1] }}; {% endif %}pvremove -fy {{ fields[0] }}; {% endfor %}"
+  shell: "{% for line in item.stdout_lines %}{% set fields = line.split() %}{% if fields | count > 1 %}lvremove -ff {{ fields[1] }}; vgremove -fy {{ fields[1] }}; vgchange -an {{ fields[1] }}; dmsetup remove {{ fields[1] }}; {% endif %}pvremove -fy {{ fields[0] }}; {% endfor %}"
```
Comment (jarrpa) [inline roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml]: Makes sense overall. Should `vgchange` come before `vgremove`?

Comment (SaravanaStorageNetwork) [reply roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml]: Yes - https://www.tldp.org/HOWTO/LVM-HOWTO/removevgs.html - updating this.

**Comment by SaravanaStorageNetwork** [comment]
@jarrpa  PTAL

**Pull request merged**